api/v1beta2/foundationdbcluster_types_test.go

```diff
@@ -4548,6 +4548,17 @@ var _ = Describe("[api] FoundationDBCluster", func() {
 				},
 				fmt.Errorf("storage engine ssd-rocksdb-v1 is not supported on version 6.3.2"),
 			),
+			Entry("using invalid storage engine",
```

Comment:
This test needs some adjustments in https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/api/v1beta2/foundationdb_version.go#L184.